sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/AnalyzeSentimentBatchDocuments.java

```diff
@@ -37,9 +37,9 @@ public static void main(String[] args) {
 
         // The texts that need be analyzed.
         List<TextDocumentInput> documents = Arrays.asList(
-            new TextDocumentInput("A", "The hotel was dark and unclean. I wouldn't recommend staying there.", "en"),
-            new TextDocumentInput("B", "The restaurant had amazing gnocchi! The waiters were excellent.", "en"),
-            new TextDocumentInput("C", "The hotel was dark and unclean. The restaurant had amazing gnocchi!", "en")
+            new TextDocumentInput("A", "The hotel was dark and unclean. I wouldn't recommend staying there."),
```

Comment:
Not sure I understand why we are hiding the language here. It is true that `en` is the default, but the purpose of this batch sample is to show the user that hey can set the language (otherwise they'll use the convenience method/sample).
It will be more obvious if we have text in different languages.

Reply:
Since the language is not required. TextAnalyticsInput currently made it as settable. So user can set the language after the object been initialized. 


It is like 
```
TextDocumentInput input1 = new TextDocumentInput("id", "text");
input1.setLanguage("en");
TextDocumentInput input2 = new TextDocumentInput("id2", "text2");
input2.setLanguage("en");

...
```

Reply:
I agree with @maririos. 
Since the batch samples show advanced behavior, we should consider showing the users how they can set language on each document.

Reply:
@samvaity is there a better to do set the language other than 
```
TextDocumentInput input1 = new TextDocumentInput("id", "text");
input1.setLanguage("en");
TextDocumentInput input2 = new TextDocumentInput("id2", "text2");
input2.setLanguage("en");
...
```